Compile fails (VNL)

I tried compiling with gcc 5.4 and 4.9, and I get this error both times:
```
[  2%] Building CXX object Modules/ThirdParty/VNL/src/vxl/vcl/CMakeFiles/itkvcl.dir/vcl_deprecated.cxx.o
In file included from /home/reckbo/soft/UKFTractography-build/ITKv4/Modules/ThirdParty/VNL/src/vxl/vcl/vcl_iosfwd.h:8:0,
                 from /home/reckbo/soft/UKFTractography-build/ITKv4/Modules/ThirdParty/VNL/src/vxl/vcl/vcl_iostream.h:27,
                 from /home/reckbo/soft/UKFTractography-build/ITKv4/Modules/ThirdParty/VNL/src/vxl/vcl/vcl_deprecated.cxx:4:
/home/reckbo/soft/UKFTractography-build/ITKv4/Modules/ThirdParty/VNL/src/vxl/vcl/vcl_compiler.h:130:4: error: #error "Dunno about this gcc"
 #  error "Dunno about this gcc"
    ^
```

--- a/SuperBuild/External_ITKv4.cmake
+++ b/SuperBuild/External_ITKv4.cmake
@@ -27,7 +27,7 @@ if(NOT DEFINED ITK_DIR AND NOT ${CMAKE_PROJECT_NAME}_USE_SYSTEM_${proj})
   endif()
 
   set(${proj}_REPOSITORY ${git_protocol}://itk.org/ITK.git)
-  set(${proj}_GIT_TAG 120b3420a9edc6488aa8f91bc8e657fb619fcabe ) # 2015-04-28 Fixed VTKGlue 
+  set(${proj}_GIT_TAG 3932df3dd174194eacf637cd3049d08809ee2bd2 ) # v4.11.0
   set(EXTERNAL_PROJECT_OPTIONAL_CMAKE_CACHE_ARGS)
 
   if(NOT ${CMAKE_PROJECT_NAME}ITKV3_COMPATIBILITY AND CMAKE_CL_64)

--- a/SuperBuild/External_SlicerExecutionModel.cmake
+++ b/SuperBuild/External_SlicerExecutionModel.cmake
@@ -33,7 +33,7 @@ if(NOT DEFINED SlicerExecutionModel_DIR AND NOT ${CMAKE_PROJECT_NAME}_USE_SYSTEM
   ExternalProject_Add(${proj}
     ${${proj}_EP_ARGS}
     GIT_REPOSITORY "${git_protocol}://github.com/Slicer/SlicerExecutionModel.git"
-    GIT_TAG "b31b424a2e18fa7e9ada11433a16b1e148d6cdd8"
+    GIT_TAG "62d0121dbb0fb057ebbd7c9ab84520accacec8bc"
     SOURCE_DIR ${CMAKE_BINARY_DIR}/${proj}
     BINARY_DIR ${proj}-build
     CMAKE_ARGS -Wno-dev --no-warn-unused-cli

